R/run.R

```diff
@@ -1,6 +1,9 @@
 
 run_r <- function(options) {
 
+  if (!file.exists(env_file))
+    stop("Unable to find environment file in temporary directory. Try restarting R session.")
+
```

Comment:
I think we should put this in the setup phase somewhere, so that it is also used for the background processes.

Reply:
@gaborcsardi I moved the check to `default_load_hook()` because that function is called by both `r()` and `r_bg()`. If there is a better place for it, please let me know.